cli/tests/unit/http_test.ts

```diff
@@ -73,6 +73,37 @@ Deno.test({ permissions: { net: true } }, async function httpServerBasic() {
   await promise;
 });
 
+// https://github.com/denoland/deno/issues/15107
```

Comment:
It's racy so not able consistently reproduce this on `main`. It fails sometimes in debug build.